index.html

```diff
@@ -271,19 +270,26 @@ <h3 class="connect">LET'S CONNECT</h3>
               </li>
               <hr>
             </ul>
-            
-            
-
-            
-            
-              
-
-                  
-                  
-                
-                  
+              <!-------------------------------------- Contact form------------------------------->
+            <div class="about-contact">
+              <div class="title-container">
+                <h2 class="contact">Contact Me</h2>
+                <p class="para">If you have an application you are interested in<br>developing, 
+                  a feature that you need built or a project<br>that needs coding. i'd love to help with it</p>
+              </div>
+              <div class="form-container">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="labels">
```

Comment:
- [ ] You need to add the `id` to the formspree to make it submit successfully. use this [Link](https://formspree.io/html/) to see the process of how to do so 👍 
- Also capitalize the `POST` method to make it a standard highlight understandable to all 👍